README.md

```diff
@@ -577,6 +642,17 @@ The allocated device information is exported in Container's environment variable
 For example, if 2 devices are allocated from `intel.com/sriov` extended resource then the allocated device information will be found in following env variable:
 `PCIDEVICE_INTEL_COM_SRIOV=0000:03:02.1,0000:03:04.3`
 
+There is also another environment variable that expose in json format more information about the device that was allocated like mount and addition variables if requested.
+The variable name is `PCIDEVICE_<RESOUCE_NAME>_INFO` appended with full extended resource name (e.g. intel.com/sriov etc.) which is capitailzed and any special characters (".", "/") are replaced with underscore ("_")
+
+vfio device example:
+```json
+PCIDEVICE_INTEL_COM_DPDK_NIC_1_INFO={"0000:3b:02.6":{"vfio":{"vfio-dev-mount":"/dev/vfio/169","vfio-mount":"/dev/vfio/vfio"},"vhost":{"net-mount":"/dev/vhost-net","tun-mount":"/dev/net/tun"}}}
+
+# Adding also the additionalInfo will inject them for example
```

Comment:
suggested reword:

> Adding also the additionalInfo

Adding `additionalInfo` field to the resource definition will add additional information for every device allocated. e.g: